dev/devicelab/bin/tasks/run_release_test.dart

```diff
@@ -2,161 +2,9 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'dart:async';
-import 'dart:convert';
-import 'dart:io';
-
-import 'package:flutter_devicelab/framework/devices.dart';
 import 'package:flutter_devicelab/framework/framework.dart';
-import 'package:flutter_devicelab/framework/task_result.dart';
-import 'package:flutter_devicelab/framework/utils.dart';
-import 'package:path/path.dart' as path;
+import 'package:flutter_devicelab/tasks/run_tests.dart';
 
 void main() {
-  task(() async {
-    final Device device = await devices.workingDevice;
-    await device.unlock();
-    final Directory appDir = dir(path.join(flutterDirectory.path, 'dev/integration_tests/ui'));
-    await inDirectory(appDir, () async {
-      final Completer<void> ready = Completer<void>();
-      final List<String> stdout = <String>[];
-      final List<String> stderr = <String>[];
-
-      // Uninstall if the app is already installed on the device to get to a clean state.
-      print('uninstalling...');
-      final Process uninstall = await startProcess(
-        path.join(flutterDirectory.path, 'bin', 'flutter'),
-        <String>['--suppress-analytics', 'install', '--uninstall-only', '-d', device.deviceId],
-      )..stdout
-        .transform<String>(utf8.decoder)
-        .transform<String>(const LineSplitter())
-        .listen((String line) {
-        print('uninstall:stdout: $line');
-      })..stderr
-        .transform<String>(utf8.decoder)
-        .transform<String>(const LineSplitter())
-        .listen((String line) {
-        print('uninstall:stderr: $line');
-        stderr.add(line);
-      });
-      if (await uninstall.exitCode != 0) {
-        throw 'flutter install --uninstall-only failed.';
-      }
-
-      print('run: starting...');
-      final Process run = await startProcess(
-        path.join(flutterDirectory.path, 'bin', 'flutter'),
-        <String>['--suppress-analytics', 'run', '--release', '-d', device.deviceId, 'lib/main.dart'],
-        isBot: false, // we just want to test the output, not have any debugging info
-      );
-      int? runExitCode;
-      run.stdout
-        .transform<String>(utf8.decoder)
-        .transform<String>(const LineSplitter())
-        .listen((String line) {
-          print('run:stdout: $line');
-          if (
-            !line.startsWith('Building flutter tool...') &&
-            !line.startsWith('Running "flutter pub get" in ui...') &&
-            !line.startsWith('Initializing gradle...') &&
-            !line.contains('settings_aar.gradle') &&
-            !line.startsWith('Resolving dependencies...') &&
-            // Catch engine piped output from unrelated concurrent Flutter apps
-            !line.contains(RegExp(r'[A-Z]\/flutter \([0-9]+\):')) &&
-            // Empty lines could be due to the progress spinner breaking up.
-            line.length > 1
-          ) {
```

Comment:
I removed these filters as they weren't necessary. This test ignores unmatched lines: it only checks that the expected lines are in standard output in the correct order.